site/content/kapp/docs/develop/apply.md

```diff
@@ -67,6 +67,18 @@ Possible values: "" (default).
 
 Annotation value will be replaced with a unique ID on each deploy. This allows to force resource update as value changes every time.
 
+### kapp.k14s.io/renew-duration
+
+Available in v0.54.0+
+
+`kapp.k14s.io/renew-duration` annotation allows specifying an interval at which resource will get renewed. 
```

Comment:
"annotation allows to specify a time interval. If `kapp deploy` is run after this interval has lapsed, kapp will force update the resource irrespective of no changes to the resources configuration. The update strategy(add link to update strategy) associated with the resource will be used."
Please specify an example